VAMobile/src/screens/auth/LoginScreen/LoginScreen.tsx

```diff
@@ -34,8 +34,10 @@ const LoginScreen: FC = () => {
   const startAuth = useStartAuth()
   const theme = useTheme()
   const [demoPromptVisible, setDemoPromptVisible] = useState(false)
-  const TAPS_FOR_DEMO = 20
+  const TAPS_FOR_DEMO = 7
+  const DEMO_TAPS_TIMEOUT_MS = 500
```

Comment:
Are there any a11y concerns with adding this timeout to access demo mode? My concern is that someone who can't tap as fast might hit the timeout and be unable to access demo mode. Also is adding this timeout a requirement mentioned in the ticket?

Reply:
No requirement in the ticket for timeout. As for a11y for timeout, I'm not sure, is there someone I can ask about that?

As for timeout, my thinking was that we don't want people accidentally triggering the demo-mode prompt, but that was purely an assumption on my part that that may be a desired way to handle it. It can be easily removed or duration could be extended. Which would be preferred?

Reply:
I would just drop this, and I feel comfortable making that call solo without leverage accessibility etc.